code/modules/events/electrical_storm.dm

```diff
@@ -65,9 +64,21 @@
 		if(prob((0.2 * severity) - 0.2))
 			T.set_broken()
 
-/datum/event/electrical_storm/end()
-	..()
+/datum/event/electrical_storm/announce_end()
 	for (var/zlevel in affecting_z)
 		if(zlevel in current_map.station_levels)
 			command_announcement.Announce("The [location_name()] has cleared the electrical storm. Please repair any electrical overloads.", "Electrical Storm Alert", zlevels = affecting_z)
 			break
+
+
+/datum/event/electrical_storm/overmap/announce_start()
+	if(affecting_shuttle)
+		send_sensor_message("Entering electrical storm.")
+		return
```

Comment:
Why not just do `Entering [name]` on the parent? You're using affecting_shuttle here anyway